sgx_main.c

```diff
@@ -184,6 +184,15 @@ static int sgx_pm_suspend(struct device *dev)
 	return 0;
 }
 
+static void sgx_reset_pubkey_hash(void *failed)
+{
+	if (wrmsrl_safe(MSR_IA32_SGXLEPUBKEYHASH0, 0xa6053e051270b7acULL) ||
```

Comment:
Why not declaring these constants as macros (like the MSRs addresses) and use them here?

Reply:
Didn't bother doing it as it is used only here :-) The names are declared because some kernels may have them